src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
```

Comment:
Move CONSTANTS outside the function.
CONSTANT - This is a variable that is created `only 1 time` and `never changes`.
In ur case, it is created every time the function is called